start-workers-mac.sh

```
@@ -84,7 +85,11 @@ activeCreds.forEach((cred, index) => {
   
   // dart install でビルドされた AOT バイナリを直接実行
   const binaryPath = '/Users/admin/Library/Application Support/Dart/install/bin/openci_worker';
-  const cmd = \`OPENCI_PROJECT_ID=\"\${projectId}\" OPENCI_API_KEY=\"\${apiKey}\" OPENCI_SERVER_URL=\"\${serverUrl}\" OPENCI_EMAIL=\"\${email}\" OPENCI_PASSWORD=\"\${password}\" '\${binaryPath}' --supervised\`;
+  let cmd = `OPENCI_PROJECT_ID=\"\${projectId}\" OPENCI_API_KEY=\"\${apiKey}\" OPENCI_SERVER_URL=\"\${serverUrl}\" OPENCI_EMAIL=\"\${email}\" OPENCI_PASSWORD=\"\${password}\" `;
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)